lib/drivers/medtronic600Driver.js

```diff
@@ -23,6 +23,11 @@ const sundial = require('sundial');
 const crypto = require('crypto');
 const iconv = require('iconv-lite');
 const crcCalculator = require('../crc.js');
+const TZOUtil = require('../TimezoneOffsetUtil');
+const lzo = require('../lzo');
+const NGPUtil = require('../medtronic600/NGPUtil');
+const NGPHistoryParser = require('../medtronic600/NGPHistoryParser');
+const Medtronic600Simulator = require('../medtronic600/medtronic600Simulator');
```

Comment:
Is it maybe possible to have a single simulator for both 5/7-series and 6-series, or are there 6-series specific things that you already had to modify the simulator for?